### PR TITLE
Feature#14-Navigation-between-screens

### PR DESCRIPTION
**Pull request by gadd0609**
navigation between screens and implementation of drawer menu

### What?
-  Drawer menu and navigation implementation.
- See Initializing the React Native project #27  for more information.
### Why
- In order for there to be navigation between the screens. 
- See Initializing the React Native project #27  for more information.
## Testing
- Since the project is just initialized you only need install the basic packages to visualize the application
```npm install```
- then you can initialize with an android emulator or via web
```npm start```
## Screenshots
| Proposed design   |      Successful design      |
|:----------:|:-------------:|
| image| image |

### CODE REVIEW

**Review by garzo94 — APPROVED** [review]

File: src/views/MyRecords.jsx [hunk 1]
```diff
@@ -13,10 +13,10 @@ export class MyRecords extends Component {
...
-          <Cards img={cardImg} />
-          <Cards img={cardImg} />
-          <Cards img={cardImg} />
-          <Cards img={cardImg} />
+          <Cards img={cardImg} onPress={() => {this.props.navigation.navigate("Details")}} />
+          <Cards img={cardImg} onPress={() => {this.props.navigation.navigate("Details")}}/>
+          <Cards img={cardImg} onPress={() => {this.props.navigation.navigate("Details")}}/>
+          <Cards img={cardImg} onPress={() => {this.props.navigation.navigate("Details")}}/>
         </ScrollView>
       </View>
```
Comment (IvanGodinez21) [inline src/views/MyRecords.jsx:21]: Next time, try to use [FlatList](https://reactnative.dev/docs/flatlist), I know that it's just a preview, but you can pass an array with any elements 😶‍🌫️

**Review by IvanGodinez21 — APPROVED** [review]

File: src/views/MyRecords.jsx (hunk 1 above)
Comment (Zuack55) [reply src/views/MyRecords.jsx:21]: thank you for the suggestion !!

File: babel.config.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
-module.exports = function(api) {
+module.exports = function (api) {
   api.cache(true);
   return {
     presets: ['babel-preset-expo'],
+    plugins: ['react-native-reanimated/plugin'],
   };
-};
+};
```
Comment (YilKanda) [inline babel.config.js:7]: Add a blank line

**Review by YilKanda — APPROVED** [review]
LGTM!

**Review by McFly1208 — APPROVED** [review]
LGTM

**Pull request merged**